.env example

```diff
@@ -0,0 +1,2 @@
+ADMIN_USERNAME=
+ADMIN_PASSWORD=
```

Comment:
@sdkaraarslan having new lines at the end of the file is pretty standard. Can you please configure your prettier/ide to add new lines on save? 

A quick search brought me this one: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files